client/store/destinationStore.js

```diff
@@ -6,7 +6,11 @@ export const processGeoJSON = createAsyncThunk(
   "destination/processGeoJSON",
   async (data, { rejectWithValue }) => {
     try {
-      const response = await axios.post(`${api}/osm/process/geojson`, data);
+      const token = await AsyncStorage.getItem('userToken');
```

Comment:
We should just add a middleware to our store like this, would cut down on a top of repeated code. Rough idea:

`const authMiddleware = (storeAPI) => (next) => (action) => {
  if (action.type.includes('pending')) {
    const token = storeAPI.getState().auth.token;

    if (token) {
      // Attach token to headers
      action.meta.arg.headers = { ...action.meta.arg.headers, Authorization: `Bearer ${token}` };
    }
  }

  return next(action);
};
`